### DISCUSSION TITLE
Scroll Bar Color

### DISCUSSION THREAD

**Discussion by andrewkhartley** [saadeghi/daisyui#2297] [Q&A] [answered]
This is a simple question, but I want to understand how to get the scrollbar to pick up the correct color scheme for light or dark mode when I create a custom theme. That's all: thank you!

**Comment by saadeghi** [comment, marked as answer]
You can use `color-scheme: light` or `color-scheme: dark` CSS rules  

In `tailwind.config.js`  
```js
module.exports = {
  plugins: [require('daisyui')],
  daisyui: {
    themes: [
      {
        mytheme: {
          'color-scheme': 'dark',  // 👈 here
          primary: '#e58b82',
          secondary: '#e83ad0',
          accent: '#45c697',
          neutral: '#372b3b',
          'base-100': '#3c3f44',
        },
      },
    ],
  },
}
```

**Reply by andrewkhartley** [reply]
Simple and effective: thank you!
